### PR TITLE
fix: SVG scaling issues in Internet Explorer

### LINKED ISSUE #6019: Overlapping graphs on overview tab when using svg graph type
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for  bugs, please do not post issues asking for help or how to do X, Y or Z. 
You can use our irc channel ##librenms on freenode to ask questions or our [community site](https://community.librenms.org).
If you have a feature request please post this on our [community site](https://community.librenms.org/c/feature-requests).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [x] Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
- [x] Please provide ALL info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [x] Please provide as much detail as possible.
- [x] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Component | Version
--------- | -------
LibreNMS  | 3f5bf56b557668659e32909204da3a42c6adac09
DB Schema | 170
PHP       | 7.0.15-0ubuntu0.16.04.2
MySQL     | 10.0.29-MariaDB-0ubuntu0.16.04.1
RRDTool   | 1.5.5
SNMP      | NET-SNMP 5.7.3

( Desktop resolution: 2560x1440 )
Decided to try out the new svg graphs, and since reading the "No doubt some small issues will arise, please let us know if that is the case." I figured I'd report this.

The graphs on overview tab are wider than normal causing them to overlap. (after ctrl+f5)
 
Edit: After testing some more it seems to be specific to IE. In Chrome & normal Firefox the graphs looks normal.

Overlapping graphs

### PR DESCRIPTION
**Pull request by geordish**
fixes #6019 

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by laf** [comment]
What's changed in the svg images?

**Comment by geordish** [comment]
Its added in a viewbox tag to those that are missing them.

https://sarasoueidan.com/blog/svg-coordinate-systems/

File: html/css/dark.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
+.ie9 img {
+    width: 100%;
+}
+@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+    img {
+        width: 100%;
```
Comment (murrant) [inline html/css/dark.css]: This makes all logos 250px wide on IE...

Comment (geordish) [inline html/css/dark.css]: Yes, I'm not convinced that this PR is the best way to go about fixing it.

I might remove the CSS changes, and leave the additions of viewbox to the SVGs. I think those are going to be required regardless.

Comment (murrant) [inline html/css/dark.css]: Yeah, I did some quick searches and tried a few things and could not fix the logos on IE11.
Tried different combinations of viewbox and height/width svg properties and the css here.

**Comment by murrant** [comment]
viewBox is needed for proper scaling.  IE fucks it up when width+height are set...

**Comment by laf** [comment]
Are the 90+ other files that have been changed supposed to be there?

**Comment by murrant** [comment]
Yes, those are the changes to fix IE, or remove svgs containing pngs.

**Pull request merged**